src/item_location.h

```diff
@@ -14,8 +16,10 @@ class vehicle_cursor;
  * Ensures the item exists, but not that the character/vehicle does.
  * Should not be kept, but removed before the end of turn.
```

Comment:
What are your intentions with this class? If it's intended to just be a serializable reference to an item (which I'd prefer) it should be documented as such, and lose the pervasive lookups of uid (it should only so a uid lookup when being deserialized), if you want it to be a stable reference to an item, it should detect and report invalidations such as the underlying item being moved or destroyed.